services/fileServices/MdPageServices/__tests__/ContactUsPageService.spec.js

```diff
@@ -0,0 +1,171 @@
+const {
+  contactUsContent: mockContactUsContent,
+  contactUsSha: mockContactUsSha,
+  rawContactUsContent: mockRawContactUsContent,
+} = require("@fixtures/contactUs")
+const {
+  footerContent: mockFooterContent,
+  footerSha: mockFooterSha,
+} = require("@fixtures/footer")
+const { NotFoundError } = require("@root/errors/NotFoundError")
+
+describe("ContactUs Page Service", () => {
+  const siteName = "test-site"
+  const accessToken = "test-token"
+  const reqDetails = { siteName, accessToken }
+
+  const CONTACT_US_FILE_NAME = "contact-us.md"
+  const CONTACT_US_DIRECTORY_NAME = "pages"
+
+  const mockFrontMatter = {
+    ...mockContactUsContent.frontMatter,
+    feedback: mockFooterContent.feedback,
+  }
+  const mockContent = mockContactUsContent.pageBody
+
+  const mockGithubService = {
+    create: jest.fn(),
+    read: jest.fn(),
+    update: jest.fn(),
+    delete: jest.fn(),
+  }
+
+  const mockFooterYmlService = {
+    read: jest.fn(),
+    update: jest.fn(),
+  }
+
+  jest.mock("@utils/markdown-utils", () => ({
+    retrieveDataFromMarkdown: jest.fn().mockReturnValue({
+      frontMatter: mockFrontMatter,
+      pageContent: mockContent,
+    }),
+    convertDataToMarkdown: jest.fn().mockReturnValue(mockRawContactUsContent),
+  }))
+
+  const {
+    ContactUsPageService,
+  } = require("@services/fileServices/MdPageServices/ContactUsPageService")
+  const service = new ContactUsPageService({
+    gitHubService: mockGithubService,
+    footerYmlService: mockFooterYmlService,
+  })
+
+  const {
+    retrieveDataFromMarkdown,
+    convertDataToMarkdown,
+  } = require("@utils/markdown-utils")
```

Comment:
could we shift the imports up to the top of the file?

Reply:
This is a little tricky because the methods need to be imported after the mocks have been established - and the mocks use variables defined only inside the test description. We could move the variable definition upwards as well - but that might cause the imports section to become messy as well. What do you think?

Reply:
oh i think that won't be an issue. this is because jest will [automatically hoist](https://jestjs.io/docs/manual-mocks#using-with-es-module-imports) ESM import mocks. 

this means that if we convert this to an import, it'll work fine. (not sure if this is the case for `require` but can try also cos TS is compiling down to ESM anyway

Reply:
As discussed offline - this unfortunately causes issues, and we're opting to leave it as is for now as it should be resolved by a typescript migration.